examples/fluids/src/smartsim/sgs_dd_training.c

```diff
@@ -311,16 +318,20 @@ PetscErrorCode TSMonitor_SGS_DD_Training(TS ts, PetscInt step_num, PetscReal sol
     {
       const PetscScalar *training_data;
       PetscCall(VecGetArrayRead(TrainingData, &training_data));
+      PetscCall(PetscLogEventBegin(SmartRedis_Train, 0, 0, 0, 0));
       PetscSmartRedisCall(put_tensor(smartsim->client, array_key, array_key_len, (void *)training_data, sgs_dd_train->training_data_array_dims, 2,
-                                     SRTensorTypeDouble, SRMemLayoutContiguous));
+                                SRTensorTypeDouble, SRMemLayoutContiguous));
+      PetscCall(PetscLogEventEnd(SmartRedis_Train, 0, 0, 0, 0));
       PetscCall(VecRestoreArrayRead(TrainingData, &training_data));
     }
-    PetscCall(SmartRedisVerifyPutTensor(smartsim->client, array_key, array_key_len));
+    //PetscCall(SmartRedisVerifyPutTensor(smartsim->client, array_key, array_key_len));
```

Comment:
Any reason not to verify that the tensor was placed correctly?
```suggestion
    PetscCall(SmartRedisVerifyPutTensor(smartsim->client, array_key, array_key_len));
```

Reply:
It would save some time, that's all, but if you think it should be there I can uncomment that check.

Reply:
For consistency with other put_tensor calls that were being verified, I kept this and added timers around it

Reply:
Fine by me. Doubt it'll matter _that_ much.